python/ray/train/examples/tune_cifar_torch_pbt_example.py

```diff
@@ -110,22 +138,25 @@ def train_func(config):
     # Create loss.
     criterion = nn.CrossEntropyLoss()
 
-    results = []
-    for _ in range(epochs):
+    while True:
         train_epoch(train_loader, model, criterion, optimizer)
         result = validate_epoch(validation_loader, model, criterion)
-        session.report(result)
-        results.append(result)
+        checkpoint = Checkpoint.from_dict(
```

Comment:
```suggestion
        checkpoint = TorchCheckpoint.from_dict(
```

Reply:
I used a generic `Checkpoint.from_dict` rather than `TorchCheckpoint.from_model()` or `TorchCheckpoint.from_state_dict()` because there's no way to save the momentum optimizer state when going through the framework specific checkpoints.

Also, doing `TorchCheckpoint.from_dict` is a bit weird because I'm not setting the model state dict to the correct key, so I wouldn't be able to do `checkpoint.get_model()` later.